src/sage/combinat/diagram_algebras.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# sage.doctest: needs sage.combinat sage.modules
 r"""
```

Comment:
remaining warnings:
```
sage -t --long --warn-long 33.8 --random-seed=61144299382637325659043318932957598596 src/sage/combinat/diagram_algebras.py
**********************************************************************
File "src/sage/combinat/diagram_algebras.py", line 2416, in sage.combinat.diagram_algebras.PartitionAlgebra
Warning: Consider using a block-scoped tag by inserting the line 'sage: # needs sage.symbolic' just before this line to avoid repeating the tag 4 times
    q = var('q')                                                              # optional - sage.symbolic
**********************************************************************
File "src/sage/combinat/diagram_algebras.py", line 2428, in sage.combinat.diagram_algebras.PartitionAlgebra
Warning: Consider using a block-scoped tag by inserting the line 'sage: # needs sage.symbolic' just before this line to avoid repeating the tag 4 times
    P = PA.basis().list()                                                     # optional - sage.symbolic
**********************************************************************
File "src/sage/combinat/diagram_algebras.py", line 2458, in sage.combinat.diagram_algebras.PartitionAlgebra
Warning: Consider using a block-scoped tag by inserting the line 'sage: # needs sage.symbolic' just before this line to avoid repeating the tag 6 times
    S = SymmetricGroupAlgebra(SR, 2)                                          # optional - sage.symbolic
**********************************************************************
File "src/sage/combinat/diagram_algebras.py", line 3839, in sage.combinat.diagram_algebras.BrauerAlgebra.jucys_murphy
Warning: Consider using a block-scoped tag by inserting the line 'sage: # needs sage.symbolic' just before this line to avoid repeating the tag 4 times
    z = var('z')                                                          # optional - sage.symbolic
    [953 tests, 16.68 s]
```